frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -61,6 +63,8 @@ const TranslationTable = ({
     setStoryTranslationContentId(storyTranslationContentId);
   };
 
+  const showStatusColumn = !isTest || isReviewable;
```

Comment:
the reason for this condition is bc we want to show the status column during the test grading (review), but not the test translation